src/app.js

```diff
@@ -1,9 +1,17 @@
 const express = require('express');
-
+const mongoose = require('mongoose');
+const user = require('./models/users')
 const app = express()
 
-app.post('/signup', (req, res) => {
-    res.send('ok')
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json())
+
+mongoose.connect(process.env.MONGO_URI).catch(error=>console.log(error.message))
+
+app.post('/signup', async(req, res) => {
```

Comment:
This route does not account for any error handling, you must specify a way for your app to catch errors

Use `try` and `catch` statements to handle errors cleverly.

See how express can help you set up error handling here https://expressjs.com/en/guide/error-handling.html